Assets/MixedRealityToolkit/_Core/Definitions/Devices/ControllerState.cs

```diff
@@ -18,10 +18,14 @@ public enum ControllerState
         /// </summary>
         Tracked,
         /// <summary>
-        /// The controller is currently not tracked and has relative positioning.
+        /// The controller is currently not tracked.
         /// </summary>
         NotTracked,
         /// <summary>
+        /// The controller is currently only returning orientation data.
+        /// </summary>
+        OrientationOnly,
```

Comment:
What about position only?

Reply:
I can add that. I was also planning on moving Other above Tracked so that the value does not change.